webhook/resources/backup/mutator.go

```diff
@@ -50,6 +53,35 @@ func (b *backupMutator) Create(request *admission.Request, newObj runtime.Object
 	}
 	patchOps = append(patchOps, patchOp)
 
+	backupLabels := make(map[string]string)
+	volumeName, isExist := backup.Labels[types.LonghornLabelBackupVolume]
+	if !isExist {
+		err := errors.Wrapf(err, "failed to ger volume name from backup label")
+		return nil, werror.NewInvalidError(err.Error(), "")
+	}
+	volume, err := b.ds.GetVolumeRO(volumeName)
+	if err != nil {
+		err := errors.Wrapf(err, "failed to get volume info: %v", volumeName)
```

Comment:
```suggestion
		err := errors.Wrapf(err, "failed to get volume %v", volumeName)
```

Reply:
Thank you. I will remove the redundant word.

Reply:
@weizhe0422 Need to resolve the code first before resolving the comment if it's not yet. Others, u will forget the suggestion instead.

Reply:
@innobead Yes, I've revised the first version and updated the backup label in mutating webhook function in my second commit.

Reply:
Thanks :)

I think your previous reply confusing me, because I thought u didn't resolve it yet but resolved the comment.

Reply:
Sorry for the confusion.